Point_set_processing_3/doc/Point_set_processing_3/Point_set_processing_3.txt

```diff
@@ -314,6 +314,296 @@ points in the domain.
+
+\cgalFigureBegin{Point_set_processing_3figOpenGR_parameter_overlap,super4PCS_overlap.png}
+The effect of varying overlap parameter on the size of the basis used for registration. The overlap is smaller for left (a) than right (b).
+\cgalFigureEnd
+
+Usually, the larger the overlap, the faster the algorithm.
+When the overlap is unknown, a simple way to set this parameter is to start from 
+100% overlap, and decrease the value until obtaining a good result.
+Using too small values will slow down the algorithm, and
+reduce the accuracy of the result.
+
+\subsubsection Point_set_processing_3Examples_registration_OpenGR_parameter_maximum_running_time Parameter: maximum_running_time
+
+Maximum number of seconds after which the algorithm stops. Super4PCS explores the
+transformation space to align the two input clouds. Since the exploration is performed
+randomly, it is recommended to use a large time value to explore the whole space.
+
+\subsection Point_set_processing_3Examples_registration_PointMatcher PointMatcher Example
+
+The following example reads two point sets and aligns them using the 
+\ref thirdpartylibpointmatcher library, using the ICP algorithm. It also shows how
+to customize ICP algorithm by using possible configurations:
+\cgalExample{Point_set_processing_3/registration_with_pointmatcher.cpp}
+
+\cgalFigureRef{Point_set_processing_3tableRegistrationRegistration_visualization_table} demonstrates
+visualization of a scan data before and after different registration methods are applied,
+including the PointMatcher registration method. To obtain the results for PointMatcher registration
+in the visualization table, abovementioned example was used.
+
+\subsubsection Point_set_processing_3Examples_registration_PointMatcher_parameter_point_set_filters Parameter: point_set_filters
+
+The chain of filters to be applied to the point cloud. The
+point cloud is processed into an intermediate point cloud with the given chain
+of filters to be used in the alignment procedure. The chain is organized with
+the forward traversal order of the point set filters range.
+
+The chain of point set filters are applied only once at the beginning of the
+ICP procedure, i.e., before the first iteration of the ICP algorithm.
+
+The filters can have several purposes, including but are not limited to:
+
+- removal of noisy points which render alignment of point clouds difficult,
+
+- removal of redundant points so as to speed up alignment,
+
+- addition of descriptive information to the points such as a surface normal vector or the direction from the point to the sensor.
+
+In registration, there are two point clouds in consideration, one of which is the reference point
+cloud while the other one is the point cloud that a transformation is computed to register that point
+cloud to the reference. The point set filters corresponds to `readingDataPointsFilters` configuration module of \ref thirdpartylibpointmatcher
```

Comment:
while the other one is the point cloud that a transformation is computed to register that point
cloud to the reference -> while the other one is the point cloud to register.